app/src/main/res/values/strings.xml

```diff
@@ -548,4 +550,5 @@
     <string name="notification_subject_release">Release</string>
     <string name="notification_interval">Sync interval</string>
     <string name="detach">Open in new window</string>
+    <string name="no_reply_group_selected_hint">Select comment group to reply to</string>
```

Comment:
Makes sense, I'll change this one too. Reason for the change, in case you were wondering, was avoiding line wrapping of the hint which I saw happening even in the Pixel emulator.